### PR TITLE
Add cluster limits for OpenShift 4.1

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This commit:
- Updates the limits page for OpenShift 4.1 based on the large scale
  test run.
- Adds a note about the single dimentional analysis we used for
  determining/validating some of the limits.

### CODE REVIEW

**Comment by chaitanyaenr** [comment]
@ahardin-rh @akrzos @ashishkamra @mffiedler PTAL.

File: modules/how-to-plan-your-environment-according-to-cluster-limits.adoc
Comment (ahardin-rh) [inline modules/how-to-plan-your-environment-according-to-cluster-limits.adoc]: Add [IMPORTANT] or [NOTE] before the first ==== to make it  a note box. You could also make this message part of the IMPORTANT note box just before it to avoid having 2 boxes in a row.

**Comment by ahardin-rh** [comment]
@chaitanyaenr Just one comment from me. Otherwise, looks good! :+1:

**Comment by chaitanyaenr** [comment]
We hit some etcd issues at 9000 projects when running the cluster limits tests to validate the namespaces per cluster limit (10,000). Here are the links to the filed BZ's - https://bugzilla.redhat.com/show_bug.cgi?id=1703581, https://bugzilla.redhat.com/show_bug.cgi?id=1706103. One of the issues was that the etcd space quota was too low and we were able to get past that after running defragmentation and bumping up the quota. The other issue was that the cluster went down when one of the etcd was down and also etcd was using lots of system resources at the same time. 

We tested this on the beta4 build and I believe that the recent builds have the fixes for most of the issues.

**Comment by chaitanyaenr** [comment]
@ahardin-rh added [Important] before the recommendation like you suggested, thanks for the review.

File: modules/how-to-plan-your-environment-according-to-cluster-limits.adoc
Comment (akrzos) [inline modules/how-to-plan-your-environment-according-to-cluster-limits.adoc]: I think the verbage here should change:
```
Some of the limits are stretched only in a single dimension, so they might vary
when a lot of objects are running on the cluster.
```
to
```
Some of the limits are stretched only in a single dimension.  They will vary
when many objects are running on the cluster.
```

**Comment by akrzos** [comment]
> We hit some etcd issues at 9000 projects when running the cluster limits tests to validate the namespaces per cluster limit (10,000). Here are the links to the filed BZ's - https://bugzilla.redhat.com/show_bug.cgi?id=1703581, https://bugzilla.redhat.com/show_bug.cgi?id=1706103. One of the issues was that the etcd space quota was too low and we were able to get past that after running defragmentation and bumping up the quota. The other issue was that the cluster went down when one of the etcd was down and also etcd was using lots of system resources at the same time.
> 
> We tested this on the beta4 build and I believe that the recent builds have the fixes for most of the issues.

I think we need to drop the namespace limit to what was achieved although the build did not have the fixes that might make this achievable.

**Comment by ahardin-rh** [comment]
@vikram-redhat This maps to https://jira.coreos.com/browse/OSDOCS-42 

@chaitanyaenr Once we get QE approval from @mffiedler we can get this merged in. Thanks!

**Comment by chaitanyaenr** [comment]
Bumped down the namespaces per cluster limit to 8000 until we test the previous limit on a large scale cluster again with the fixed build.

File: modules/openshift-cluster-limits.adoc
Comment (warmchang) [inline modules/openshift-cluster-limits.adoc]: Is it really 250? Not 2500?

Comment (chaitanyaenr) [reply modules/openshift-cluster-limits.adoc]: @warmchang we lowered the value to 250 as it's the maximum node scale we tested for OCP 4.1. We can bump it up to 2000 if and when we get an opportunity to run tests at a higher node count.

Comment (warmchang) [reply modules/openshift-cluster-limits.adoc]: @chaitanyaenr Got it, thank you for reply!

**Comment by ahardin-rh** [comment]
@mffiedler PTAL

File: modules/openshift-cluster-limits.adoc
Comment (mffiedler) [inline modules/openshift-cluster-limits.adoc]: nit:  "When there are a large number of active projects" 
spelling:  "maintenance"   Suggested:  "Periodic maintenance of etcd, including defragmentation, is highly recommended to free storage in etcd"

**Review by mffiedler — CHANGES_REQUESTED** [review]

File: modules/openshift-cluster-limits.adoc
Comment (chaitanyaenr) [reply modules/openshift-cluster-limits.adoc]: Fixed, thanks.

File: modules/openshift-cluster-limits.adoc
Comment (mffiedler) [inline modules/openshift-cluster-limits.adoc]: I think you missed my first one:   s/When there are large number/When there are a large number/

File: modules/openshift-cluster-limits.adoc
Comment (ahardin-rh) [reply modules/openshift-cluster-limits.adoc]: Thanks! Merging! If we need further edits, we can add them in a separate PR.

Comment (ahardin-rh) [reply modules/openshift-cluster-limits.adoc]: Spoke too soon. See Mike's latest comment.

**Review by mffiedler — APPROVED** [review]
/lgtm

File: modules/openshift-cluster-limits.adoc
Comment (chaitanyaenr) [reply modules/openshift-cluster-limits.adoc]: Fixed, thanks.

**Comment by chaitanyaenr** [comment]
Thanks for the review.

**Pull request merged**
